index.html

```diff
@@ -1 +1,34 @@
-<!DOCTYPE html>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
The indentation for the whole file looks a bit odd. Can you make sure that you are using spaces instead of tabulations? Thanks